### ISSUE TITLE
The latest available Docker image for ARM is v0.38

### ISSUE THREAD

**Issue by rogatty** [zadam/trilium#902]
See https://hub.docker.com/r/hlince/trilium/tags
I guess the build automation (https://gitea.e9g.rocks/howard/trilium-daily-build) failed?

**Comment by zadam** [comment]
Yes, looks like that it's not up to date. However, these are unofficial builds and I'm not involved in managing them.

**Comment by rogatty** [comment]
I understand these are not official builds 🙂 Any chance you have contact with Howard? He seems to not be here on GitHub or I'd ping him directly.

**Comment by simonmeulenbeek** [comment]
For an easy workaround, you can git clone the repository and change the first line of the `Dockerfile` to be `FROM --platform=arm node:12.16.3-alpine`.
Then run `docker build .`

I've just tested it on my RPI, and it works flawlessly.

**Comment by bryanasdev000** [comment]
> For an easy workaround, you can git clone the repository and change the first line of the `Dockerfile` to be `FROM --platform=arm node:12.16.3-alpine`.
> Then run `docker build .`
> 
> I've just tested it on my RPI, and it works flawlessly.

Works like a charm. Thanks!

**Comment by onjour** [comment]
> > For an easy workaround, you can git clone the repository and change the first line of the `Dockerfile` to be `FROM --platform=arm node:12.16.3-alpine`.
> > Then run `docker build .`
> > I've just tested it on my RPI, and it works flawlessly.
> 
> Works like a charm. Thanks!

Sadly I couldn't make it work. 

It fails to build because of 

`COPY failed: file not found in build context or excluded by .dockerignore: stat server-package.json: file does not exist`

I don't know what file I'm supposed to use to generate this, can you help or provide yours? 

Thank you

**Comment by YC** [comment]
> It fails to build because of
> 
> `COPY failed: file not found in build context or excluded by .dockerignore: stat server-package.json: file does not exist`
> 
> I don't know what file I'm supposed to use to generate this, can you help or provide yours?

I encountered the same issue. 
It seems to build correctly after `cp package.json server-package.json`.
